src/app/player/components/player.component.html

```diff
@@ -1,3 +1,8 @@
+<div class="player-container" *ngIf="numberOfVictories$ | async as numberOfVictories">>
+  <ng-container *ngIf="numberOfVictories > 0">
```

Comment:
just `numberOfVictories` should suffice I think

Reply:
I know, I put it in purpose for readability, do you want it badly to be changed?

Reply:
Don't we have it like this in other parts of the code? In my opinion it is more readable and just more elegant, and shorter makes it always easier to understand. I will not insist, I already approved, but I will try to shame you into doing it.

Reply:
Here you go :)